addons/website/data/ir_asset.xml

```diff
@@ -0,0 +1,168 @@
+
+            <record id="assets_snippet_s_masonry_block_css_000_c2" model="ir.asset">
+                <field name="name">assets snippet s masonry block css 000</field>
+                <field name="bundle">web.assets_frontend</field>
+                <field name="directive">append</field>
+                <field name="glob">website/static/src/snippets/s_masonry_block/000.scss</field>
+                <field name="target" eval="None"></field>
+                <field name="active" eval="False"></field>
+                <field name="sequence" eval="16"></field>
+            </record>
+
+            <record id="assets_snippet_s_rating_css_000_43" model="ir.asset">
+                <field name="name">assets snippet s rating css 000</field>
+                <field name="bundle">web.assets_frontend</field>
+                <field name="directive">append</field>
+                <field name="glob">website/static/src/snippets/s_rating/000.scss</field>
+                <field name="target" eval="None"></field>
+                <field name="active" eval="False"></field>
+                <field name="sequence" eval="16"></field>
+            </record>
+
+            <record id="assets_snippet_s_title_css_000_8b" model="ir.asset">
+                <field name="name">assets snippet s title css 000</field>
+                <field name="bundle">web.assets_frontend</field>
+                <field name="directive">append</field>
+                <field name="glob">website/static/src/snippets/s_title/000.scss</field>
+                <field name="target" eval="None"></field>
+                <field name="active" eval="False"></field>
+                <field name="sequence" eval="16"></field>
+            </record>
+
+            <record id="assets_snippet_s_image_gallery_css_000_05" model="ir.asset">
+                <field name="name">assets snippet s image gallery css 000</field>
+                <field name="bundle">web.assets_frontend</field>
+                <field name="directive">append</field>
+                <field name="glob">website/static/src/snippets/s_image_gallery/000.scss</field>
+                <field name="target" eval="None"></field>
+                <field name="active" eval="False"></field>
+                <field name="sequence" eval="16"></field>
+            </record>
+
+            <record id="assets_snippet_s_three_columns_css_000_78" model="ir.asset">
+                <field name="name">assets snippet s three columns css 000</field>
+                <field name="bundle">web.assets_frontend</field>
+                <field name="directive">append</field>
+                <field name="glob">website/static/src/snippets/s_three_columns/000.scss</field>
+                <field name="target" eval="None"></field>
+                <field name="active" eval="False"></field>
+                <field name="sequence" eval="16"></field>
+            </record>
+
+            <record id="assets_snippet_s_showcase_css_000_e5" model="ir.asset">
```

Comment:
- Why not leaving those in the file where we defined them? We made a huge refactoring last year to be able to find all information about a snippet from the directory in which they are defined, I don't see a reason for this to change :wink: 
- What is the strange 2 letters you added at the end of each asset XML ID? :thinking: 
- As previously explained, I would have liked to keep the possibility to extend `website.assets_frontend` and not `web.assets_frontend` even if they are the same bundle (see previous explanation for details why). I see you changed that to web.assets_frontend everywhere here so I wonder... :grimacing: 
- Is repeating "target None, sequence 16" everywhere actually needed?

Reply:
@qsm-odoo Sorry I didn't know we had to put it as draft when it's not meant to be merged :) But indeed, it's still WIP. 
But your input is welcome: 

_Why not leaving those in the file where we defined them? We made a huge refactoring last year to be able to find all information about a snippet from the directory in which they are defined, I don't see a reason for this to change 😉_

The entire conversion is done by a script that we wrote. So when we need to define an ir.asset manually (because some attribute must be changed and can't be done in the manifest), we put them all in this file. **BUT** I've discovered that the only places where it is needed is in three website modules. So we can definitely move them back where you wish them to be. 

_What is the strange 2 letters you added at the end of each asset XML ID? 🤔_

I've been told that id must be unique (Right ?), and collision could occur. Once again, now that I know that only website is concerned, that can be safely dropped. 

_As previously explained, I would have liked to keep the possibility to extend website.assets_frontend and not web.assets_frontend even if they are the same bundle (see previous explanation for details why). I see you changed that to web.assets_frontend everywhere here so I wonder... 😬_

I don't think I'm aware of this conversation ? Do you have a link to this so I get your motivation on this point ? :) The solution most likely will be a new primary bundle website.assets_frontend that includes web.assets_frontend then. 

_Is repeating "target None, sequence 16" everywhere actually needed?_

Nope, just the lazy way is the script was to keep them all 😬